src/main/java/com/codejoust/main/dto/game/GameMapper.java

```diff
@@ -84,9 +85,22 @@ public static SubmissionDto submissionToDto(Submission submission) {
             return null;
         }
 
+        List<SubmissionResultDto> notHidden = new LinkedList<>();
```

Comment:
I could be missing something, but wondering why you chose a `LinkedList` rather than an `ArrayList` here?

Reply:
There was no real reason. I thought it would be marginally faster, but I can change it to ArrayList if desired.